src/locales/translations/pt.ts

```diff
@@ -8,89 +8,83 @@ export default {
   minimize: "Minimizar",
   maximize: "Maximizar",
   close: "Fechar",
-  add_server_modal_description_1:
-    "Adicione servidores manualmente à lista de favoritos.",
+  add_server_modal_description_1: "Adicionar servidor à lista de favoritos.",
   add_server_modal_description_2: "Exemplo: 127.0.0.1:7777",
   add: "Adicionar",
-  server: "Servidores",
+  server: "Servidor",
   address: "Endereço IP",
-  players: "Jogadores",
+  players: "Players",
   server_join_prompt_enter_password:
-    "Este servidor está trancado, por favor digite a senha.",
-  server_join_prompt_enter_password_input_placeholder: "Digite a senha...",
-  server_join_prompt_nickname_input_placeholder: "Digite o Nome de Usuário...",
+    "Este servidor está trancado, insira a senha",
+  server_join_prompt_enter_password_input_placeholder: "Insira a senha...",
+  server_join_prompt_nickname_input_placeholder: "Insira o nome de usuário...",
   connect: "Conectar",
-  copy: "Copiar",
+  copy: "COPIAR",
   remove_from_favorites: "Remover dos favoritos",
   add_to_favorites: "Adicionar aos favoritos",
-  filters: "Filtros",
-  filter_only_omp_servers: "Somente servidores open.mp",
+  filters: "Filters",
```

Comment:
Overall, the review is good. However, I didn't understand why "Filtros" was changed to "Filters" here. It doesn't make sense, since both have the same number of characters, and more importantly, "Filters" isn't a valid word in Portuguese.

Reply:
My mistake, i used the english file! Sorry, will review it again

Reply:
Should be updated now

Reply:
Yes, like "Apenas servidores open.mp" sounds more European than Brazillian but it avoids the text cutoff issue. But anything, just suggest what to change and i'll review it and commit,